app/views/products/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<nav>
```

Comment:
It seems like this `nav` section should be moved out into a layout and/or partial